tests/cypress/support/commands.js

```diff
@@ -88,28 +88,55 @@ Cypress.Commands.add('openTaskJob', (taskName, jobNumber=0) => {
     cy.openJob(jobNumber)
 })
 
-Cypress.Commands.add('createShape', (firstX, firstY, lastX, lastY) => {
+Cypress.Commands.add('createShape', (firstX, firstY, secondX, secondY, points='By 2 Points',
+                                     thirdX, thirdY, fourthX, fourthY, switchLabel=false, labelName) => {
     cy.get('.cvat-draw-rectangle-control').click()
+    if (switchLabel) {
+        cy.get('.cvat-draw-shape-popover-content')
+        .find('.ant-select-selection-selected-value')
+        .click()
+        cy.get('.ant-select-dropdown-menu')
+        .contains(labelName)
+        .click()
+    }
```

Comment:
Let's organize it as a different command to switch a label

Reply:
Done.